package.json

```diff
@@ -6,16 +6,16 @@
   "scripts": {
     "start": "node src/index.js",
     "dev": "nodemon src/index.js",
-    "deploy": "node src/deployCommands.js"
+    "deploy": "node src/deployCommands.js",
+    "syncdb": "node src/syncdb.js"
   },
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@discordjs/builders": "^1.4.0",
-    "@discordjs/rest": "^1.4.0",
-    "discord-api-types": "^0.37.21",
-    "discord.js": "^14.7.1",
+    "discord.js": "^14.9.0",
```

Comment:
upgrading dependencies here isn't optimal, since it has nothing to do with what this PR is about (see "atomic commits"). but might as well keep it now, as long as it didn't break anything

Reply:
(less important than the other 2, mostly just a nitpick)